### PR TITLE
A R Sandbox  

### PR DESCRIPTION
**Pull request by shubhammalviya06**
Pull request for ar based project documentation

### CODE REVIEW

**Comment by agshubhs** [comment]
@shubhammalviya06 
1. Video spelling is wrong in a folder name.
2. Space should be there between "Mechanical Design" in the folder name.
3. Make the table for cost structure.
4. Add the team image and good images in the folder as well as readme file.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+
+## ABSTRACT:
+
+A R Based Map Projector A R Sandbox is hands-on sandbox combined with 3 D visualisation applications where sand is augmented in real-time with an elevation color map, topographic contour lines, and simulated water.Simply mold the sand by hand and the landscape comes to life! The AR Sandbox uses a 3D camera such as the Microsoft Kinect to scan the 3D shape of the sand surface in real time.A computer collects data from the camera, and creates a real-time dynamic topography map using the Processing software program. A projector, also mounted above the sandbox, projects that topography map back onto the real sand, calibrated such that real and virtual features line up exactly.
+
+## Motivation
+Aerial Mapping and surveying.
+
+## Mechanical Aspect of the Design:
+1. Sandbox of dimensions(75*60cm)
```
Comment (agshubhs) [inline README.md:16]: First letter of each line should be capital

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
...
+## Cost Structure:
+- Kinect:Rs 5000
+- Projector:Rs 22000
+- white sand: (10 kg) Rs 100
+- (nut,bolt,aluminium rod,L joints): Rs 300
+
+
+## Applications:
+
+- dynamic projection
```
Comment (agshubhs) [inline README.md]: This should be wriiten in numbered list

**Comment by agshubhs** [comment]
For images you can use Sristhi Drive link.
Two same phots are there in two folders

**Comment by agshubhs** [comment]
@shubhammalviya06  
Insert your own picture Do not use the pictures from the internet, so remove the first picture.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,92 @@
 # AR-Map-Based-Projector
 Srishti 2019
+
+## Image of the project
+
+<img src="Images%20and%20Videos/Images/ARSandbox.png" width="400" >
```
Comment (agshubhs) [inline README.md]: Remove this picture and take the picture from  Srishti Drive Link
Used Markdown Syntax in place of HTML.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,92 @@
...
+
+## Cost Structure:
+
+|components  |    cost       |
+|------------|-------------| 
+| Kinect           |5000 Rs   | 
+| Projector           |22000 Rs| 
+| white sand           |500 Rs|
+|nut,bolt,aluminium rod,L joints|300 Rs|
+
```
Comment (agshubhs) [inline README.md:43]: Add total column in the table also

File: README.md [hunk 5]
```diff
@@ -1,2 +1,92 @@
...
+<img src="https://user-images.githubusercontent.com/48958222/55109353-abbfd500-50fb-11e9-9ba8-3227992f6d11.jpeg" width="250">
+
+
+## Mentors:
+1. Kunal Kumar
+2. Shreeyash Geda
+
+## References
+For more information, visit[ AR Sandbox](https://arsandbox.ucdavis.edu/)
```
Comment (agshubhs) [inline README.md]: remove this link instead  of this post the video on youtube and paste that link of your project.

**Comment by shubhammalviya06** [comment]
Send me shristi drive Link

**Comment by agshubhs** [comment]
@shubhammalviya06 
1. Increase the size of the images as they ar too small. you can use HTML syntax for image.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,92 @@
...
+## Softwares used
+1. kinect SDK
+2. KinectConfigurationVerifier
+3. processing-3.5.3
+4. Kinect Pv2 library
+
+
+## Cost Structure:
+
+|S.No|components  |    cost       |
```
Comment (agshubhs) [inline README.md:37]: 1. The first letter of each word should be capital.
2.  Write Cost(INR) in place of Cost Add a total column in the Table.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,92 @@
...
+
+<img src="https://user-images.githubusercontent.com/48958222/55109353-abbfd500-50fb-11e9-9ba8-3227992f6d11.jpeg" width="250">
+
+
+## Mentors:
+1. Kunal Kumar
+2. Shreeyash Geda
+
+## References
+For more information, visit[ AR Sandbox](https://www.youtube.com/watch?v=YLYO0YhY83w)
```
Comment (agshubhs) [inline README.md:78]: Remove this link

**Comment by agshubhs** [comment]
@shubhammalviya06  
Rename the images according to their description.

**Comment by agshubhs** [comment]
@shubhammalviya06 
Rename images according to the description

**Comment by agshubhs** [comment]
@shubhammalviya06  Write  Cost(INR) in table in first row rather writing rs in each row.

**Pull request merged**